EventFilter/HcalRawToDigi/plugins/HcalRawToDigi.cc

```diff
@@ -59,6 +61,25 @@ HcalRawToDigi::HcalRawToDigi(edm::ParameterSet const& conf):
 // Virtual destructor needed.
 HcalRawToDigi::~HcalRawToDigi() { }  
 
+void HcalRawToDigi::fillDescriptions(edm::ConfigurationDescriptions& descriptions) {
+  edm::ParameterSetDescription desc;
+  desc.addUntracked<int>("HcalFirstFED",int(FEDNumbering::MINHCALFEDID));
+  desc.add<int>("firstSample",0);
+  desc.add<int>("lastSample",9);
+  desc.add<bool>("FilterDataQuality",true);
+  desc.addUntracked<std::vector<int>>("FEDs", std::vector<int>());
+  desc.addUntracked<bool>("UnpackZDC",true);
+  desc.addUntracked<bool>("UnpackCalib",true);
+  desc.addUntracked<bool>("UnpackTTP",true);
+  desc.addUntracked<bool>("silent",false);
```

Comment:
this looks like a bug
https://github.com/perrotta/cmssw/blob/multithreadEventFilter/EventFilter/HcalRawToDigi/plugins/HcalRawToDigi.cc#L24
has
```C++
 silent_(conf.getUntrackedParameter<bool>("silent",true)),
```